workflow/sample_files/base-appliances-coal.xml

```diff
@@ -113,7 +113,6 @@
             <SolarAbsorptance>0.7</SolarAbsorptance>
             <Emittance>0.92</Emittance>
             <Pitch>6.0</Pitch>
-            <RadiantBarrier>false</RadiantBarrier>
```

Comment:
Do we want to preserve `false` here?

Reply:
I'm okay with these optional elements being omitted.